### PR TITLE
Fix the cluster terminology for v5

### PR DESCRIPTION
**Pull request by NataliaIvakina**
Replace 'Causal Cluster' by 'Cluster'.
Change link to the Kafka Connector manual on the docs page.
Some small tweaks in different sections.

### CODE REVIEW

File: modules/ROOT/pages/cypher-intro/results.adoc [hunk 1]
```diff
@@ -732,7 +732,7 @@ RETURN p.name, size(collect(friend.name)) AS numberOfFriends;
 .Query5 Results:
 image:cypher_agg_size.jpg[role="popup-link"]
 
-In Neo4j 5.0, if you need to find a number of relationship patterns, you should use the `COUNT {}` expression.
+In Neo4j 5, if you need to find a number of relationship patterns, use the `COUNT {}` expression.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/cypher-intro/results.adoc:735]:
```suggestion
In Neo4j v5, if you need to find a number of relationship patterns, use the `COUNT {}` expression.
```

File: modules/ROOT/pages/get-started-with-neo4j/index.adoc [hunk 2]
```diff
@@ -7,7 +7,7 @@ There are a number of options on how to get started with Neo4j, and your choice
 [[neo4j-first-steps]]
 == First steps with Neo4j
 
-1. If you want to learn Neo4j graph database features, how to write Cypher queries, use developer tools like xref:get-started-with-neo4j/graph-platform.adoc#neo4j-tools[Neo4j Browser and Bloom], and get an overview on how to connect to Neo4j via drivers and connectors, go to the link:https://neo4j.com/cloud/platform/aura-graph-database/[AuraDB page], where you can create a new Neo4j instance free forever. +
+1. If your purpose is to learn Neo4j graph database features, how to write Cypher queries, use developer tools like xref:get-started-with-neo4j/graph-platform.adoc#neo4j-tools[Neo4j Browser and Bloom] and get an overview on how to connect to Neo4j via drivers and connectors, go to the link:https://neo4j.com/cloud/platform/aura-graph-database/[AuraDB page], where you can create a Neo4j instance free forever. +
```
[the remark this answers is not in the document]
Comment (AlexicaWright) [reply modules/ROOT/pages/get-started-with-neo4j/index.adoc:10]: Also, unsure if this is true? Is the source of this link really the place we want to send people?

File: modules/ROOT/pages/get-started-with-neo4j/index.adoc [hunk 3]
```diff
@@ -7,7 +7,7 @@ There are a number of options on how to get started with Neo4j, and your choice
...
 AuraDB is the best choice for cloud developers who are interested in graph technologies. +
```
Comment (AlexicaWright) [inline modules/ROOT/pages/get-started-with-neo4j/index.adoc:11]:
```suggestion
AuraDB is a good choice for cloud developers who are interested in graph technologies. +
```

File: modules/ROOT/pages/languages-guides/index.adoc [hunk 4]
```diff
@@ -27,14 +27,13 @@ You should have link:{aura_signup}[created an Neo4j AuraDB cloud instance], or l
...
-To build an application, you want to connect to Neo4j from your technology stack.
-Fortunately it is very easy using a driver which connects to Neo4j via Bolt or HTTP.
+You are able to build an application, you want to connect to Neo4j from your technology stack, using a driver which connects to Neo4j via Bolt or HTTP.
 
 [#bolt-protocol]
 == The binary Bolt Protocol
 
-Starting with Neo4j 3.0 we support a binary protocol called Bolt.
-It is based on the PackStream serialization and supports the Cypher type system, protocol versioning, authentication and TLS via certificates.
+Starting with Neo4j 3.0, we support a binary protocol called Bolt.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/languages-guides/index.adoc:35]:
```suggestion
Starting with Neo4j v3.0, we support a binary protocol called Bolt.
```

File: modules/ROOT/pages/languages-guides/index.adoc [hunk 5]
```diff
@@ -27,14 +27,13 @@ You should have link:{aura_signup}[created an Neo4j AuraDB cloud instance], or l
...
+It is based on the PackStream serialization and supports the Cypher type system, protocol versioning, authentication, and TLS via certificates.
 For Neo4j Clusters, Bolt provides smart client routing with load balancing and failover.
 
 The binary protocol is enabled in Neo4j by default, so you can use any language driver that supports it.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/languages-guides/index.adoc:39]: Maybe a link to the Bolt docs somewhere? 
https://neo4j.com/docs/bolt/current/

File: modules/ROOT/pages/languages-guides/index.adoc [hunk 6]
```diff
@@ -60,21 +59,21 @@ Get started with your stack now, see the dedicated page for more detail.
 
 [cols="5*",width=100]
```
Comment (AlexicaWright) [inline modules/ROOT/pages/languages-guides/index.adoc:60]: A comment for line 56, I suggest changing to:
Thanks to the Neo4j contributor community, there are additional drivers for almost every popular programming language, most of which mimic existing database driver idioms and approaches.

File: modules/ROOT/pages/languages-guides/java/java-intro.adoc [hunk 7]
```diff
@@ -15,10 +15,10 @@ While this guide is not comprehensive, it introduces the different APIs and link
 [abstract]
 You should be familiar with xref:appendix/graphdb-concepts/index.adoc[graph database concepts] and the property graph model.
 You should have link:{aura_signup}[created an Neo4j AuraDB cloud instance], or link:/download/[installed Neo4j locally].
-When developing with Neo4j, please use Java 8 or 11 and your favorite IDE.
+When developing with Neo4j, use Java 8 or 11 and your favorite IDE.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/languages-guides/java/java-intro.adoc:18]:
```suggestion
When developing with Neo4j, use Java 8 or 11 and your preferred IDE.
```

**Review by AlexicaWright — COMMENTED** [review]
A few comments and suggestions for improvement. Thank you @NataliaIvakina !

File: modules/ROOT/pages/get-started-with-neo4j/index.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (NataliaIvakina) [reply modules/ROOT/pages/get-started-with-neo4j/index.adoc:10]: https://neo4j.com/cloud/platform/aura-graph-database/ is the Aura product  page. You see subscription plans there. Button 'Start Free' is one of the first things that grab your attention

[the remark this answers is not in the document]
Comment (AlexicaWright) [reply modules/ROOT/pages/get-started-with-neo4j/index.adoc:10]: Right, my concern is whether this is the best place to start?

File: modules/ROOT/pages/languages-guides/index.adoc (hunk 5 above)
Comment (NataliaIvakina) [reply modules/ROOT/pages/languages-guides/index.adoc:39]: good idea! thank you

File: modules/ROOT/pages/get-started-with-neo4j/index.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (NataliaIvakina) [reply modules/ROOT/pages/get-started-with-neo4j/index.adoc:10]: I think , yes. What entry-point to AuraDB could you recommend?

[the remark this answers is not in the document]
Comment (AlexicaWright) [reply modules/ROOT/pages/get-started-with-neo4j/index.adoc:10]: Let's discuss this afternoon during our workshop.

**Review by AlexicaWright — APPROVED** [review]
Looks good! I'd say merge when you want. Thank you @NataliaIvakina

**Pull request merged**